src/migration/1744972478032-SpecialUserDeleteTrigger.ts

```diff
@@ -0,0 +1,42 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class SpecialUserDeleteTrigger1744972478032
+  implements MigrationInterface
+{
+  name = 'SpecialUserDeleteTrigger1744972478032';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `DROP RULE prototect_ghostuser_deletion on "user";`,
+    );
+    await queryRunner.query(`
+      CREATE OR REPLACE FUNCTION prevent_special_user_delete()
```

Comment:
Instead of `RULE` create trigger, because `RULE` was blocking usage of `RETURNING` in zombie cron.